Doc/library/inspect.rst

```diff
@@ -702,6 +699,13 @@ function.
       Python.  For example, in CPython, some built-in functions defined in
       C provide no metadata about their arguments.
 
+   .. note::
```

Comment:
Perhaps this is more fitting.

```suggestion
   .. impl-detail::
```

Reply:
I like making this distinction @erlend-aasland.